synthea/README.md

```diff
@@ -0,0 +1,75 @@
+- You may also use the flexporter standalone to map an existing exported file `./run_flexporter -fm {mapping_file_location} -s {source_fhir_file}`
+- See the [flexporter documentation](https://github.com/synthetichealth/synthea/wiki/Flexporter) for additional information on the flexporter, mapping file, and limitations.
+
+## Building the Mapping File
+Quality Measurement calculation requires data conformant with qicore, and expansive IG, which would require expansive effort to fully map. As such, we can piecemeal address the IG requirements by supporting requirements for individual measures. The recommended process for creating a mapping that addresses a set of measures is:
+
+1. Use [elm-parser-for-ecqms fhir_review branch](https://github.com/projecttacoma/elm-parser-for-ecqms/tree/fhir_review) and get data requirements to build a combined list of mustSupports for all resources across the set of measures.
+2. Use [fqm-execution](https://github.com/projecttacoma/fqm-execution) and get data requirements to build a full list of profiles used.
+3. For all resource types, ensure resource is exported as a top level resource by Synthea or already supported by the flexporter mapping. If not, use the flexporter `create_resource` action to export the resource based on a logical existing exported resource or based on a logical Synthea module state. Resources that are only used for SDEs may be initially ignored.
+3. Make sure all profiles are applied to the correct exported resource.
+4. For each must support:
+    - Check the resource's [qicore 4.1.1](https://hl7.org/fhir/us/qicore/STU4.1.1/) profile Snapshot Table to check if the mustSupport is also required by the profile (minimum cardinality 1). If not, it may be initially ignored.
+    - If required, check if synthea already exports it by looking in the [FhirR4.java](https://github.com/synthetichealth/synthea/blob/master/src/main/java/org/mitre/synthea/export/FhirR4.java) file. You can look for a string like `Condition()` for an example of how the Fhir object is created and what fields are set on it.
+    - If Synthea doesn't export it, check if the existing mapping file already adds it through a mapping.
+    - If not, fill from something logical that synthea does export (if something logical exists). Understanding what values are logical may involve looking at the FHIR specification and/or the measure's CQL for how the must support field is used in the measure logic.
+    - If a logical field doesn't exist, fill from a chosen pre-set value or choose randomly from a limited set of values 
+
+## CMS130 Mapping Example
+
+Below is the example list of resources and must support elements collated from elm-parser-for-ecqms data requirement calculation. Unexported values are marked with a `*` with notes about how their export was resolved in the `qicore_130.yaml` mapping file.
+
+- Condition:
+  - "code",
+  - "clinicalStatus",
+  - "onset",
+  - "abatement"
```

Comment:
Synthea does export abatement, but only if the modules cause the condition to end (i.e. abate). Seems like that didn't happen in the runs you did - it may be a less common module state to end the condition.
Here's how synthea does it: https://github.com/synthetichealth/synthea/blob/055d977ec40f52cd5d386395297930241ad4e34c/src/main/java/org/mitre/synthea/export/FhirR4.java#L1651